src/bootstrap/src/core/download.rs

```diff
@@ -616,8 +616,6 @@ impl Config {
         };
 
         // For the beta compiler, put special effort into ensuring the checksums are valid.
-        // FIXME: maybe we should do this for download-rustc as well? but it would be a pain to update
-        // this on each and every nightly ...
```

Comment:
It sounds painful and I think we will never be doing this.

Reply:
For a future possibility, maybe distribute the hashes with the tarballs?
Or resurrect the signing of the artifacts...